packages/workflow/src/workflow.ts

```diff
@@ -902,13 +902,16 @@ export class Workflow {
 		// Check if there are any trigger or poll nodes and then return the first one
 		let node: INode;
 		let nodeType: INodeType;
-		for (const nodeName of nodeNames) {
-			node = this.nodes[nodeName];
 
-			if (nodeNames.length === 1 && !node.disabled) {
+		if (nodeNames.length === 1) {
+			node = this.nodes[nodeNames[0]];
+			if (!node.disabled) {
```

Comment:
Hi @iocanel thank you for the PR! If you could make this small change, I’ll be happy to approve it.

```suggestion
			if (node && !node.disabled) {
```